h/static/scripts/group-forms/components/test/EditGroupMembersForm-test.js

```diff
@@ -11,32 +11,60 @@ describe('EditGroupMembersForm', () => {
   let fakeCallAPI;
 
   beforeEach(() => {
+    const headers = { 'Misc-Header': 'Some-Value' };
     config = {
       api: {
         readGroupMembers: {
           url: '/api/groups/1234/members',
           method: 'GET',
-          headers: { 'Misc-Header': 'Some-Value' },
+          headers,
+        },
+        removeGroupMember: {
+          url: '/api/groups/1234/members/:userid',
+          method: 'DELETE',
+          headers,
         },
       },
       context: {
         group: {
           pubid: '1234',
           name: 'Test group',
         },
+        user: {
+          userid: 'acct:jane@localhost',
+        },
       },
     };
 
     fakeCallAPI = sinon.stub();
+    fakeCallAPI.rejects(new Error('Unknown API call'));
     fakeCallAPI.withArgs('/api/groups/1234/members').resolves([
       {
+        userid: 'acct:bob@localhost',
         username: 'bob',
+        actions: ['delete'],
       },
       {
+        userid: 'acct:johnsmith@localhost',
         username: 'johnsmith',
+        actions: [],
+      },
+      {
+        userid: 'acct:jane@localhost',
+        username: 'jane',
+        actions: ['delete'],
```

Comment:
Add a new user who is the current user according to `config.context.user.userid`.